tests/conftest.py

```diff
@@ -73,3 +86,20 @@ def main_page_car_hire_tab(browser):
 @pytest.fixture(scope="module")
 def main_page_search_hotels_tab(browser):
     return MainPageSearchHotelsTab(browser=browser, url=browser.current_url)
+
+
+@pytest.fixture(scope="module")
+def assertions(browser):
+    return Assertions(browser=browser)
+
+
+@pytest.fixture()
+def user(request):
+    user_cmd = request.config.getoption("user")
+    return user_cmd
+
+
+@pytest.fixture()
+def password(request):
+    password_cmd = request.config.getoption("password")
+    return password_cmd
```

Comment:
if password_cmd== "None":
        return config.DefaultCreds.PASSWORD
    else:
        return password_cmd
I consider add this

Reply:
Done